README.md

```diff
@@ -98,10 +98,9 @@ To make sure a parser is at the latest compatible version (as specified in `nvim
 
 Each module provides a distinct tree-sitter-based feature such as [highlighting](#highlight), [indentation](#indentation), or [folding](#folding); see [`:h nvim-treesitter-modules`](doc/nvim-treesitter.txt) or ["Available modules"](#available-modules) below for a list of modules and their options.
 
-All modules are disabled by default and need to be activated explicitly in your `init.vim`, e.g., via
+All modules are disabled by default and need to be activated explicitly in your `init.lua`, e.g., via
```

Comment:
I'd still say that init.vim is more common

Reply:
it is, but the following code is in lua so better to be explicit here. Should i merge ?

Reply:
would you mind referencing `:h lua-heredoc` for someone not clear on what to do with the Lua code (e.g. *"(see `:h lua-heredoc` if you're using Vimscript for init.vim)"*)